test/test_other.py

```diff
@@ -14269,6 +14269,44 @@ def test_windows_batch_file_dp0_expansion_bug(self):
     create_file('build_with_quotes.bat',  f'@"emcc" {test_file("hello_world.c")}')
     self.run_process(['build_with_quotes.bat'])
 
+  @only_windows('Check that directory permissions are properly retrieved on Windows')
+  @requires_node
+  def test_windows_nodefs_execution_permission(self):
+    if self.get_setting('WASMFS'):
```

Comment:
I think this will always be false unless you add `@also_with_wasmfs`.  I'm not sure if you want to do that here though.

Reply:
I added this part based on what I saw in other tests ([example](https://github.com/fluiddot/emscripten/blob/fa80cf25b8641974d60605b8a24cb7150fd63701/test/test_core.py#L5788-L5795)). Not exactly sure if it's needed but we could remove it if it's not 👍.